share/spice/reddit_user_search/reddit_user_search.js

```diff
@@ -0,0 +1,84 @@
+(function (env) {
+    "use strict";
+    
+     env.ddg_spice_reddit_user_search = function (api_result) {
+        // Check if we have data to work with.
+        if(api_result.error) {
+            return Spice.failed('reddit_user_search');
+        }
+        Spice.add({
+            id: "reddit_user_search",
+            name: "Social",
+            data: api_result.data,
+            meta: {
+                itemType: api_result.data.name + " (User)",
+                sourceUrl: 'http://www.reddit.com/u/' + api_result.data.name,
+                sourceName: 'Reddit'
+            },
+            templates: {
+		group: 'base',
+		options: {
+                    content: Spice.reddit_user_search.content,
+		    moreAt: true
+		}
+            }   
+        });
+    }
+}(this));
+
+Spice.registerHelper("reddit_user_search_timeConverter", function(timestamp){
+    "use strict";
+    
+    var created   = new Date(timestamp*1000),
+        months    = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'],
+        year      = created.getFullYear(),
+        month     = months[created.getMonth()],
+        date      = created.getDate(),
+        hour      = created.getUTCHours() < 10 ? '0' + created.getUTCHours() : created.getUTCHours(),
+        min       = created.getUTCMinutes() < 10 ? '0' + created.getUTCMinutes() : created.getUTCMinutes(),
+        sec       = created.getUTCSeconds() < 10 ? '0' + created.getUTCSeconds() : created.getUTCSeconds(),
+        time      = date + ' ' + month + ' ' + year + ' ' + hour + ':' + min + ':' + sec ;
+    
+  return time;
+});
+
+Spice.registerHelper("reddit_user_search_friendly_account_age", function(timestamp){
```

Comment:
We've now added support for moment.js. If you wrap the `Spice.add()` call in `DDG.require('moment.js', function() { ... });` you'll be able to use `moment()` which should reduce this function considerable!